cmake/Modules/Findaom.cmake

```diff
@@ -0,0 +1,50 @@
+# - Try to find aom
+# Once done this will define
+#
+#  AOM_FOUND - system has aom
+#  AOM_INCLUDE_DIR - the aom include directory
+#  AOM_LIBRARIES - Link these to use aom
+#
+#=============================================================================
+#  Copyright (c) 2020 Andreas Schneider <asn@cryptomilk.org>
+#
+#  Distributed under the OSI-approved BSD License (the "License");
+#  see accompanying file Copyright.txt for details.
+#
+#  This software is distributed WITHOUT ANY WARRANTY; without even the
+#  implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the License for more information.
+#=============================================================================
+#
+
+if (UNIX OR MINGW)
```

Comment:
Maybe it's better to word it this way, just remove the `if (UNIX OR MINGW)`, let the `if (PKG_CONFIG_FOUND)` handle if PkgConfig was found